.travis.yml

```diff
@@ -64,6 +70,7 @@ notifications:
 before_install:
 - pip install codecov
 - pip install numpy==1.6.1
+- pip install pyyaml
```

Comment:
It is used for reading yaml files. The package name is a bit misleading, it is used as `import yaml`.